ui/css/design-system/_colors.scss

```diff
@@ -12,6 +12,7 @@ $color-map: (
   'icon-default': --color-icon-default,
   'icon-alternative': --color-icon-alternative,
   'icon-muted': --color-icon-muted,
+  'icon-inverse': --color-icon-inverse,
```

Comment:
adding new icon tokens that were introduced in https://github.com/MetaMask/metamask-extension/pull/33764